### PR TITLE
Updated notification channels for Market

### PR DESCRIPTION
**Pull request by geoffevason**
Updated notification channels

### CODE REVIEW

**Review by johnsyweb — APPROVED** [review]
🚢

File: config/envato.yml [hunk 1]
```diff
@@ -41,44 +41,6 @@ market-conversion:
...
-    - recommendation-api
-    - api-gateway
-    - build.envato.com
-    - search_public_api
-    - discovery_users
-    - ami-builder
-    - discovery-docker
-  use_labels: true
-  exclude_labels:
-    - wip
```
Comment (jaymiejones86) [inline config/envato.yml:80]: This seems to be removing the notifications to `#market-pull-requests` for the specified repos.

Is this functionality being replaced or simply being removed as it has not proven to be affective?

**Review by jaymiejones86 — APPROVED** [review]
👍

File: config/envato.yml (hunk 1 above)
Comment (geoffevason) [reply config/envato.yml:80]: Removed intentionally. Many were duplicated in the other channels. Others are defunct.

**Pull request merged**